messagingpush/src/main/java/io/customer/messagingpush/CustomerIOPushNotificationHandler.kt

```diff
@@ -168,13 +168,9 @@ internal class CustomerIOPushNotificationHandler(
             .setStyle(NotificationCompat.BigTextStyle().bigText(body))
         tintColor?.let { color -> notificationBuilder.setColor(color) }
 
-        // This is custom logic to add images to a push notification. Some image URLs have not
-        // been able to display an image in a push. https://github.com/customerio/issues/issues/7293
-        // Instead, we recommend customers use `notification.image` in the FCM payload to bypass
-        // our logic and use the logic from FCM's SDK for image handling instead.
```

Comment:
This comment was added to [help out customers in the past](https://github.com/customerio/issues/issues/7293) where they would not see an image for *some* image URLs (the ticket has some example URLs that failed). 

[That ticket](https://github.com/customerio/issues/issues/7293) was made a while ago so there is a chance that the customer issue has been resolved and our SDK is able to show images with all URLs now. If that is true, I am OK with removing this comment. 

If our SDK still encounters problems with some image URLs, I am still OK with removing this comment as long as we create a new ticket for us to address this problem of increasing image URL compatibility in the SDK in the future.

Reply:
Your concern is very valid Levi, 

From LJ's response in the ticket
>It ended up not being an issue with the link, but instead an issue with the payload they were using to test. @levibostian any reason to keep this image open?

and my testing, I tested the very same URL you mentioned in the ticket that didn't work, 

It works:
![Screenshot_1676666457](https://user-images.githubusercontent.com/4658866/219789060-248c708d-09dd-4100-8a94-935741b18c2c.png)

And for the reasons, that we want to push people to use `Rich Push` which is `data only`, hence I decided to remove the comment. 

>we create a new ticket for us to address this problem of increasing image URL compatibility in the SDK in the future.
Would you advise creating it now or when an issue occurs? because I noticed the PR that adds it just got merged and we didn't get this issue reported for months after we reworked the image handling in SDK and sorted all the edge cases. Happy to create a ticket, if its serves a purpose.